### PR TITLE
HackerRank C++ Solutions

### PR DESCRIPTION
**Pull request by SanandhKumar02**

### CODE REVIEW

File: 3]. Competitive Programming/HackerRank/C++/Arrays Introduction.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <cmath>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/HackerRank/C++/Arrays Introduction.cpp:1]: Can You add the HackerRank C++ problem in folder
image

and put the number in front of the question like  `_01)_Hello_World.cpp` in

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Add problems in the folder
image

File: 3]. Competitive Programming/HackerRank/C++/Arrays Introduction.cpp (hunk 1 above)
Comment (SanandhKumar02) [reply 3]. Competitive Programming/HackerRank/C++/Arrays Introduction.cpp:1]: Yeah Sure, no problem. I haven't completed all the problems but I will do it ASAP.

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Comment by SanandhKumar02** [comment]
Hey,
Why i'm not being listed as a contributor?

**Pull request merged**